src/byui/cit260/sevenyearsofplenty/control/CropsControl.java

```diff
@@ -121,4 +121,31 @@ public static int harvestCrops(Crops theCropsObj) {
     } 
     
     
+
+        //reqNutrition is how much each Member of the Population needs each year
+    //to be sustained.  
+    public static int feedPeople(int reqNutrition, int feedAmount, Crops theCropsObj) {
+        int wheatInStore = theCropsObj.getWheatInStore();
+        int currPopulation = theCropsObj.getPopulation();
+        float productivity;
+
+        if (feedAmount <= 0 || wheatInStore <= 0 || currPopulation <= 0
+                || reqNutrition <= 0 || reqNutrition > feedAmount || feedAmount > wheatInStore) {
+            return -1;
+        } else {
+            theCropsObj.setWheatInStore(wheatInStore -= feedAmount);
+            int fedPopulation = feedAmount / reqNutrition;
```

Comment:
Am I reading this right? it looks like you're assuming multiple bushels will feed just one person maybe.